.github/workflows/mlir-tensorrt-ci.yml

```diff
@@ -0,0 +1,227 @@
+name: MLIR-TensorRT CI
+
+on:
+  pull_request:
+    branches:
+      - main
+    #types: [synchronize, opened, reopened, ready_for_review]
```

Comment:
Why are these commented out? I believe we do want to filter based on files changed, correct?